app/views/layouts/_flash_messages.html.slim

```diff
@@ -1,4 +1,7 @@
 - flash.each do |variant, message|
   - next unless variant.in? t("banners").keys.map(&:to_s)
   = govuk_notification_banner title_text: t("banners.#{variant}"), classes: "govuk-notification-banner--#{variant} govuk-!-margin-top-3 govuk-!-margin-bottom-0" do |notification_banner|
-    - notification_banner.with_heading(text: message)
```

Comment:
I was trying to figure out how to keep this area generic while implementing changes only in the affected area (the Jobseeker session controller, model, etc).
So, following the typical devise tweaking approach, I search what we need to override from Devise :smile: 

In this case, the message is built in `Devise::Models::Confirmable#inactive_message`. So I attempted redefining it in `Jobseeker#inactive_message` and then building up the text with the link there.

But with my approach, we have different issues: 
- We must declare the text as HTML safe in the banner layout code. 
- We are embedding HTML in the notification banner helper text parameter instead of using the parameters created for this specific purpose.
- We need to load route helpers for rails (for getting the URL) in a rails model.

Given those downsides, I give up and think your code has fewer downsides than what I've attempted.
We may not like it, but is simple enough.